metadataproxy/__init__.py

```diff
@@ -14,3 +14,10 @@
     from metadataproxy.routes import mock  # NOQA
 else:
     from metadataproxy.routes import proxy  # NOQA
+
+
+@app.before_first_request
+def setup_logging():
```

Comment:
We have a few PRs open about logging, and none of them really accomplish what I really want. I'd really like the logging to be configured through gunicorn config, not directly in the app. Can you take this out of the PR? I'll put some time in to push in a PR that addresses logging in the way I'm wanting it accomplished.